pom.xml

```diff
@@ -490,5 +490,11 @@
             <version>2.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- Used for change env system -->
+            <groupId>com.github.stefanbirkner</groupId>
+            <artifactId>system-rules</artifactId>
+            <version>1.19.0</version>
```

Comment:
Perhaps, restrict this to `test` scope.  [Docs](https://github.com/firebase/firebase-admin-java/pull/510/files?file-filters%5B%5D=.xml)

Reply:
Done!